### PR TITLE
added search bar on header

### PR DESCRIPTION
**Pull request by ilyabaranowski**

### CODE REVIEW

File: Triphouse-project/profile_active.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ghost) [inline Triphouse-project/profile_active.svg]: Put all images into folder `images`.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -35,6 +35,26 @@
...
+                            <input type="date" id="out" />
+                            <input id="qtpeople" type="text" placeholder="2 Adults — 0 Children — 1 Room"
+                                name="qtpeople" />
+
+                        </div>
+                    </div>
+                    <button>Search</button>
+                </form>
                 <div class="header__appstore-links">
                     <a href="#"><img src="../Triphouse-project/header_Gplay.svg" alt="playmarket"></a>
```
Comment (ghost) [inline Triphouse-project/index.html]:
```suggestion
<a href="#"><img src="./header_Gplay.svg" alt="playmarket"></a>
```

File: Triphouse-project/index.html [hunk 3]
```diff
@@ -35,6 +35,26 @@
...
                     <a href="#"><img src="../Triphouse-project/header_Apple.svg" alt="appstore"></a>
```
Comment (ghost) [inline Triphouse-project/index.html]:
```suggestion
   <a href="#"><img src="./header_Apple.svg" alt="appstore"></a>
```

File: Triphouse-project/profile_active.svg (hunk 1 above)
Comment (ilyabaranowski) [reply Triphouse-project/profile_active.svg]: puted all images to the folder img

File: Triphouse-project/index.html (hunk 2 above)
Comment (ilyabaranowski) [reply Triphouse-project/index.html]: done

File: Triphouse-project/index.html (hunk 3 above)
Comment (ilyabaranowski) [reply Triphouse-project/index.html]: done

**Pull request merged**